src/main/java/org/openhab/tools/analysis/checkstyle/AboutHtmlCheck.java

```diff
@@ -0,0 +1,123 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.tools.analysis.checkstyle;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.List;
+
+import org.apache.commons.io.IOUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+import org.openhab.tools.analysis.checkstyle.api.AbstractStaticCheck;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+
+/**
+ * Checks if an about.html file is valid.
+ *
+ * @author Petar Valchev
+ *
+ */
+public class AboutHtmlCheck extends AbstractStaticCheck {
+    private static final String SUSPEND_CHECKS_MSG = "No checks for the about.html file will be done.";
+
+    private static final String VALID_ABOUT_HTML_FILE_LINK_MSG = "Here is an example of a valid about.html file: https://eclipse.org/legal/epl/about.html";
```

Comment:
No configuration is ok, the problem is in the check.

I will fix it, it seems to be pretty easy to solve this.